install_multiple_targets/after/Pods/Target Support Files/Pods-SampleAppTests/Pods-SampleAppTests.debug.xcconfig

```diff
@@ -1,6 +1,6 @@
 GCC_PREPROCESSOR_DEFINITIONS = $(inherited) COCOAPODS=1
 HEADER_SEARCH_PATHS = $(inherited) "${PODS_ROOT}/Headers/Public" "${PODS_ROOT}/Headers/Public/JSONKit" "${PODS_ROOT}/Headers/Public/Reachability"
-LIBRARY_SEARCH_PATHS = $(inherited) "${PODS_CONFIGURATION_BUILD_DIR}/JSONKit" "${PODS_CONFIGURATION_BUILD_DIR}/Reachability"
+LIBRARY_SEARCH_PATHS = $(inherited) "${PODS_CONFIGURATION_BUILD_DIR}/JSONKit"
```

Comment:
why is Reachability removed from here? Especially given its on the HEADER_SEARCH_PATHS?

Reply:
I am not saying its wrong, just trying to understand it

Reply:
Because `-l"Reachability"` is not in `OTHER_LDFLAGS`, we don't need the library search path (since we're not linking the library here)